tests/test_kessel.py

```
@@ -0,0 +1,185 @@
+    builder.oauth2_client_authenticated.assert_not_called()
+
+    kessel.reset_caches()
+
+
+def test_build_credentials(mocker):
+    from roadmap.config import Settings
+
+    settings = Settings(
+        kessel_auth_oidc_issuer="https://sso.example.com/realms/redhat",
+        kessel_auth_client_id="my-client",
+        kessel_auth_client_secret="my-secret",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "kessel_auth_client_secret"

(S106)